index.html

```diff
@@ -5,25 +5,32 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Victor's Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" media="screen and (max-width: 699px)" href="style.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+    <link rel="stylesheet" media="screen and (min-width: 700px)" href="min_style_700px.css">
 </head>
 <body>
-    <header class="header">
-        <a href="#">Victor</a>
-        <nav class="navbar">          
+    <header class="header-container">
```

Comment:
Kindly fix the right spacing as the previous reviewer asked
![image](https://user-images.githubusercontent.com/24703171/159958931-a259c592-6254-4479-a55a-b5a77445ea3f.png)
The problem happens because of the header section, you can solve it by:
- [ ] Kindly remove the `margin-right: 20px` of the `hamburger` button button.
- [ ] Please use the `display: none` property for the `ul` that contains desktop links instead of `visibility: hidden` and remove `display: flex` in the mobile version.
- [ ] Please use the `display: flex` property for the `ul` that contains desktop links instead of `visibility: visible` in the desktop version.

Reply:
@gemmen29  Thanks for your review. I believe your observation and review are the best practice using ### display: none.

Attached is the screenshoot of the revision accordingly.  
![image](https://user-images.githubusercontent.com/61356487/160079945-fd4c4736-cf45-48f5-80d6-ba399ccc2a70.png)


_